.travis.yml

```diff
@@ -61,11 +63,13 @@ jobs:
       php: 7.2
       env: PHPCS_BRANCH="dev-master" WPCS_BRANCH="2.2.0" LINT=1
     - php: 7.2
-      env: PHPCS_BRANCH="3.3.1" WPCS_BRANCH="dev-develop"
+      env: PHPCS_BRANCH="3.3.1" WPCS_BRANCH="dev-master"
 
   allow_failures:
     # Allow failures for unstable builds.
     - php: "nightly"
+    - env: WPCS_BRANCH="dev-develop"
```

Comment:
If you address my first comment, you can remove this line.

Alternatively, add an extra build in the matrix to still have one test run against WPCS `develop` to know what we'll need to adjust if and when that gets released, like:
```yaml
    - php: 7.4
      env: PHPCS_BRANCH="dev-master" WPCS_BRANCH="dev-develop"
``` 

Then, have the following in `allowed_failures`:
```yaml
    - env: PHPCS_BRANCH="dev-master" WPCS_BRANCH="dev-develop"
```

Note: this has to be an _exact_ match.